bloomfilter/tests/QC.hs

```diff
@@ -33,12 +35,16 @@ tests = [
  , testProperty "Word8" $ prop_pai (undefined :: Word8)
  , testProperty "Word16" $ prop_pai (undefined :: Word16)
  , testProperty "Word32" $ prop_pai (undefined :: Word32)
- , testProperty "Word64" $ prop_pai (undefined :: Word64)
- , testProperty "String" $ prop_pai (undefined :: String)
+-}
```

Comment:
Are we expecting these to come back or should we just delete them?

If they might come back, can we leave a TODO to say here under what circumstances (e.g. would need appropriate hashable instances or something).

Reply:
They will come (or go) in ~next (cleanup) PR, so I don't feel like adding TODOs